dask_expr/_collection.py

```diff
@@ -726,6 +727,9 @@ def merge(
             Shuffle backend to use if shuffling is necessary.
         npartitions : int, optional
             The number of output partitions
+        broadcast : float, bool, optional
+            Whether to perform a broadcast merge if bool, or the broadcast bias if
```

Comment:
What's the broadcast bias?

Reply:
copied the docstring over from dask/dask now